Add content from community doc PRs (775, 787, 794, 798)

jillian-maroket opened the pull request:
Scope: v1.4, v1.5, v1.6

PRs:
- [775](https://github.com/harvester/docs/pull/775): Typo in Third-Party Storage Support file 
- [787](https://github.com/harvester/docs/pull/787): Storage network and cluster network content updates
- [794](https://github.com/harvester/docs/pull/794): VIP assigned node and MAC address
- [798](https://github.com/harvester/docs/pull/798): RWX non-root user permission issues in v1.4.3

jillian-maroket commented on versions/v1.4/modules/en/pages/networking/storage-network.adoc:
```diff
@@ -24,19 +24,19 @@ There are some prerequisites before configuring the Harvester Storage Network se
 ** `kubectl get -A vmi`
 +
-* All pods that are attached to Longhorn Volumes should be stopped.
+* All pods that are attached to {longhorn-product-name} Volumes should be stopped.
```
Yes. I'll address that in this PR.

jillian-maroket commented on versions/v1.4/modules/en/pages/networking/storage-network.adoc:
```diff
@@ -24,19 +24,19 @@ There are some prerequisites before configuring the Harvester Storage Network se
 [CAUTION]
 ====
-If the Harvester cluster was upgraded from v1.0.3, please check if Whereabouts CNI is installed properly before you move on to the next step. We will always recommend following this guide to check. https://github.com/harvester/harvester/issues/3168[Issue 3168] describes that the Harvester cluster will not always install Whereabouts CNI properly.
+If the {harvester-product-name} cluster was upgraded from v1.0.3, please check if Whereabouts CNI is installed properly before you move on to the next step. We will always recommend following this guide to check. https://github.com/harvester/harvester/issues/3168[Issue 3168] describes that the {harvester-product-name} cluster will not always install Whereabouts CNI properly.
```
The author was probably referring to the documentation in general. I'll remove this sentence. It does not add any value.

jillian-maroket commented on versions/v1.4/modules/en/pages/networking/storage-network.adoc:
```diff
@@ -71,9 +71,9 @@ kubectl apply -f https://raw.githubusercontent.com/harvester/harvester/v1.1.0/de
+* Exclude IP addresses that {longhorn-product-name} pods and the storage network must not use, such as addresses reserved for xref:/integrations/rancher/csi-driver.adoc#_rwx_volume_support[RWX volume support], the gateway, and other components.
 
-We will take the following configuration as an example to explain the details of the Storage Network
+We will take the following configuration as an example to explain the details of the storage network
```
Yes. I'll fix this.

jillian-maroket commented on versions/v1.4/modules/en/pages/networking/storage-network.adoc:
```diff
@@ -82,109 +82,145 @@ We will take the following configuration as an example to explain the details of
 ----
 
-The value format is JSON string or empty string as shown in below.
+The storage network is automatically enabled in the following situations:
```
"storage network" is a common noun. I'll check the usage of backticks in the file.

jillian-maroket commented on versions/v1.4/modules/en/pages/networking/storage-network.adoc:
```diff
@@ -82,109 +82,145 @@ We will take the following configuration as an example to explain the details of
 ==== Step 1
 
-Check if Harvester Storage Network setting's status is `True` and the type is `configured`.
+Check if storage network setting's status is `True` and the type is `configured`.
```
`storage-network` is the name of the setting.

dariavladykina commented on versions/v1.6/modules/en/pages/networking/storage-network.adoc:
```diff
@@ -82,109 +82,145 @@ We will take the following configuration as an example to explain the details of
 ==== Step 1
 
-Check if Harvester Storage Network setting's status is `True` and the type is `configured`.
+Check if storage network setting's status is `True` and the type is `configured`.
```
```suggestion
Check if the storage network setting's status is `True` and the type is `configured`.
```

jillian-maroket commented:
@ranjinimn and @dariavladykina Thank you for reviewing. I have addressed most of the issues that you flagged. 

`storage-network.adoc` contains legacy content that is not up to standard. I've been restructuring and rewriting topics like this whenever time permits.

jillian-maroket commented on versions/v1.4/modules/en/pages/networking/storage-network.adoc:
```diff
@@ -1,18 +1,18 @@
 = Storage Network
 
-Harvester uses Longhorn as its built-in storage system to provide block device volumes for VMs and Pods. If the user wishes to isolate Longhorn replication traffic from the Kubernetes cluster network (i.e. the management network) or other cluster-wide workloads. Users can allocate a dedicated storage network for Longhorn replication traffic to get better network bandwidth and performance.
+{harvester-product-name} uses {longhorn-product-name} as its built-in storage system to provide block device volumes for VMs and Pods. If the user wishes to isolate {longhorn-product-name} replication traffic from the Kubernetes cluster network (i.e. the management network) or other cluster-wide workloads. Users can allocate a dedicated storage network for {longhorn-product-name} replication traf ...
```
I addressed the issue in [this commit](https://github.com/rancher/harvester-product-docs/pull/85/commits/4f18f19792232a085f97e5bebd8760a0ff4faed7).
